lib/stove-testing-e2e-kafka/src/main/kotlin/com/trendyol/stove/testing/e2e/standalone/kafka/KafkaSystem.kt

```diff
@@ -64,8 +65,11 @@ internal fun TestSystem.kafka(): KafkaSystem = getOrNone<KafkaSystem>().getOrEls
 internal fun TestSystem.withKafka(options: KafkaSystemOptions = KafkaSystemOptions()): TestSystem {
     val kafka =
         withProvidedRegistry("confluentinc/cp-kafka:latest", options.registry) {
-            KafkaContainer(it).withExposedPorts(*options.ports.toTypedArray()).withEmbeddedZookeeper()
```

Comment:
There are two things

- the place for change should be located [`spring-e2e-kafka`](https://github.com/Trendyol/stove/blob/main/starters/spring/stove-spring-testing-e2e-kafka/src/main/kotlin/com/trendyol/stove/testing/e2e/kafka/Options.kt#L20)

- This is a behavior change, I would suggest that we should keep the previous behavior as default but we can provide  a flag `useEmbeddedZookeeper: Boolean`.

here is my suggestion:

```kotlin
@StoveDsl
data class KafkaContainerOptions(
    override val registry: String = DEFAULT_REGISTRY,
    override val image: String = "confluentinc/cp-kafka",
    override val tag: String = "latest",
    override val compatibleSubstitute: String? = null,
    val useEmbeddedZookeper: Boolean = true,
    val containerFn: KafkaContainer.() -> Unit = { },
) : ContainerOptions
```

```kotlin
.apply {
               if(options.useEmbeddedZookeper) {
                      .withEmbeddedZookeeper()
                   }
              options.containerOptionsConfigurer(this) // also enable container configuration.
           }
```

Reply:
- why we have 2 kafka client?
- exposing all configuration is better solution instead of expose single configuration because when we need or developer needs override another configuration this mean he/she needs to add feature to stove so it not sustainable.

Reply:
Standalone Kafka is an experimental project, we don't use it for now.

I am not against for exposing `KafkaContainer` if that's the question. I am okay with the implementation but there are small things to change.

Reply:
I will move to [spring-e2e-kafka](https://github.com/Trendyol/stove/blob/main/starters/spring/stove-spring-testing-e2e-kafka/src/main/kotlin/com/trendyol/stove/testing/e2e/kafka/Options.kt#L20) 
do you suggest using extra boolean field for zookeper and provide configure function (containerFn) ? Doesn't it create confusion, because we will accept boolean parameters but this parameter also can be override using  containerFn?

Reply:
It won't create I believe, `containerFn` is an advance usage scenario, people will know what to use and what not. With boolean value we keep backwards compatibility. Because without Embedded~ookeeper kafka would not start as far as I know.

Why did you need to remove it in the first place?

Reply:
according my tests kafka continues to work without embedded kafka because kafka from 2.8 is not depended to zookeeper any more.

you can see from:
https://github.com/testcontainers/testcontainers-java/blob/main/docs/modules/kafka.md#benefits

why we disabled zookeeper? because it is not required and it can be slow down boot time of kafka container (because of consensus)

Reply:
I've tested without it and it works, correct. So, I've made some changes by considering the need here, please review: https://github.com/Trendyol/stove/pull/372

Reply:
nice work, I approved.